source/hiera/3.0/release_notes.markdown

```diff
@@ -3,6 +3,13 @@ layout: default
 title: "Hiera 3: Release Notes"
 ---
 
+## Hiera 3.0.4
+
+Released October 29, 2015.
+
+Shipped in puppet-agent version 1.2.7.
+
+Hiera 3.0.4 adds support for AIX 5.3.
```

Comment:
@kylog I only see tickets for AIX 5.3. Should this include Solaris 11 and AIX 6.1/7.1?